fast-list.js

```diff
@@ -158,38 +219,35 @@ FastList.prototype = {
    * determine if we can do more expensive rendering (idle) or if we need
    * to skip rendering altogether (busy).
    *
+   * @private
    */
-  updateRenderingWindow: function(instant) {
-    var geo = this.geometry;
-
+  processScrollPosition: function(instant) {
     var position = this.els.container.scrollTop;
+    var geo = this.geometry;
 
-    // Don't compute lag on first load
-    if (geo.topPosition === -1) {
+    // Don't compute lag on first reading
```

Comment:
The first time we 'read' the scrollTop from the DOM, we don't want to compute lag.